CefSharp/DependencyChecker.cs

```diff
@@ -69,22 +69,36 @@ public static class DependencyChecker
         {
             "CefSharp.Core.dll",
             "CefSharp.dll",
-            "CefSharp.BrowserSubprocess.Core.dll",
```

Comment:
Need to also check that `CefSharp.BrowserSubprocess.Core.dll` is present for completeness.